src/qibolab/instruments/zhinst.py

```diff
@@ -799,6 +800,7 @@ def sweep_recursion(self, qubits, exp, exp_calib, relaxation_time, acquisition_t
 
         parameter = None
         print("sweep", sweeper.parameter.name.lower())
+        # print(qubits[sweeper.pulses[0].qubit])
```

Comment:
I understand this is also for debugging, but avoid as much as possible to commit dead code (i.e. if you really know you will need it again and for a while, fine, otherwise, consider keeping it in your local version, without committing to Git)

Reply:
I will just keep the AcquisitionType and AveragingMode prints